internal/idsutil/big_endian_converter.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package tracetranslator
+package idsutil
```

Comment:
It looks like this package doesn't exist currently.  Can we call it `idutils` instead?  Also, please add a `doc.go` with a brief package-level godoc describing what the package is for.  Even though this is internal that can still be useful for future developers looking to understand what it does.

Reply:
Sure. I just changed the name o `idutils` and added the `doc.go`.